content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,106 @@
+Contact Form` on your website and toggling the option on or off.
 
-You may want to know more from your visitor when they use the contact form to
-you will need to build a custom contact form on your
-website. These contact forms can generate multiple types of records in
-the system (emails, leads/opportunities, project tasks, helpdesk
-tickets, etc.)
+.. image:: generate_from_website/contact-form-toggle.png
+   :align: center
+   :alt: Contact Form toggle
+
+When the form is toggled off, the *Contact Us* page will have a message letting customers know
+that your company accepts email contact and a send now button.
 
-Configuration
-=============
+.. image:: generate_from_website/default-contact-us-page-no-form.png
+   :align: center
+   :alt: Contact Us Page using email
 
-You will need to install the free *Form Builder* module, only
-available in Odoo Enterprise.
+To change the form to a specific sales channel, go to :menuselection:`Website --> Configuration -->
+Settings.` Under *Communication,** you will find where to change which *Sales Team* or
+*Salesperson* will be assigned to any incoming leads or opportunities from your Contact Form.
 
-.. image:: media/generate_from_website05.png
+.. image:: generate_from_website/contact-form-settings.png
    :align: center
+   :alt: Contact Form Settings
 
-Create a custom contact form
-----------------------------
+Creating a custom contact form
+==============================
 
-First, go to the page where you want to put your contact form. In edit mode, drag
-the form builder onto the page, and you will be able to add all the fields
-you wish.
+Wanting more info from visitors using the contact form? Contact forms can be customized for the
+specific information your team needs because they can generate multiple types of records in the
+system, such as emails, leads/opportunities, project tasks, helpdesk tickets, and more.
 
-.. image:: media/generate_from_website06.png
+The free *Form Builder* module can be installed from either the apps page or automatically when
+added to a page in the Website Builder. Adding a whole new form can be great if you are
+soliciting certain information that needs filtering, such as suggestions, bug reports, etc.
+
+.. image:: generate_from_website/form-building-blocks.png
+   :align: center
+   :alt: Form Builder building blocks
+
+However, if editing the available predefined options is all that’s necessary, editing the Contact
+Form itself is the best option.
+
+Editing contact form fields
+---------------------------
+In edit mode, click on the field to change, and you can edit any of the following information for
+the specific field:
+
+- **Type**: Choose a custom field option or an existing field. Examples include phone, file
+  upload, language, etc.
+- **Input Type**: This determines the type of entry customers should input. Available options are
+  text, email, telephone, and URL.
```

Comment:
After "Input Type", there is the "Input Placeholder" option you didn't mention. Maybe it's on purpose, but as it seems you are going for an exhaustive description of the options, I would include it as well. (the placeholder text can be useful to guide visitors when entering info by giving them usually an example on how to fill in a field, such as a specific number phone format)